src/tests/get_released_revisions.py

```diff
@@ -84,6 +85,11 @@ def get_image_name_in_registry(img_name: str, revision: str) -> str:
 
         released_revisions[img] = []
         for risks in releases.values():
+            if risks.get("end-of-life") and risks["end-of-life"] < datetime.now(
```

Comment:
well...this is a tricky one. if there's no EOL, we could skip indeed. But I think...we shouldn't. I.e. if we don't know the EOL, the safest is to simply keep supporting it. There should be a warning for it, ofc, but it would be dismissive and possibly dangerous to skip it

Reply:
@cjdcordeiro  This will skip only if there is an EOL value and its older than current datetime.

Reply:
yes you're right. bad reading on my side.

can you pls include a warning log for cases where the eol is not set?